toolchain/semantics/semantics_handle_operator.cpp

```diff
@@ -51,7 +51,15 @@ auto SemanticsHandleInfixOperator(SemanticsContext& context,
               resume_block_id));
       return true;
     }
-
+    case TokenKind::Equal: {
+      // TODO: handle complex assignment expression such as `a = a + 1`.
```

Comment:
I would expect `a = a + 1;` already works; did you mean `a += 1;` here?

Reply:
ah yes I meant to write a+=1. It's fixed now.